src/main/java/com/alexbro/onlinebank/core/dao/user/UserDao.java

```diff
@@ -7,4 +7,6 @@
 public interface UserDao {
 
     Optional<User> getUserByLogin(String username);
```

Comment:
1."getUser" part is redundant! It is obvious that you get User from User dao.
2. You get user by username not by login!
Name of the method should be 'getByUsername'